data/registry_model/datatypes.py

```diff
@@ -331,6 +331,13 @@ def get_parsed_manifest(self, validate=True):
             self.internal_manifest_bytes, self.media_type, validate=validate
         )
 
+    @property
+    def id(self):
+        """
+        The ID of this manifest.
+        """
+        return self._db_id
```

Comment:
Other datatypes have this property, added for consistency (and so that we don't have to access `_db_id` outside the class)